cst438_gradebook/src/test/java/com/cst438/EndToEndTestAssignments.java

```diff
@@ -78,15 +78,15 @@ public void testCreateAssignment() throws Exception {
 			driver.findElement(By.xpath("//input[@name='handleSubmit']")).click();
 			Thread.sleep(SLEEP_DURATION);
 			
-			Optional<Assignment> a = assignmentRepository.findByCourseIdAndName(TEST_COURSE_ID, TEST_ASSIGNMENT_NAME);
+			Optional<Assignment> a = assignmentRepository.findAssignmentByCourseIdAndName(TEST_COURSE_ID, TEST_ASSIGNMENT_NAME);
```

Comment:
Doesn't return Optional<Assignment> anymore 

```suggestion
			List<Assignment> a = assignmentRepository.findAssignmentByCourseIdAndName(Integer.parseInt(TEST_COURSE_ID), TEST_ASSIGNMENT_NAME);
```